benchmark/README.md

```diff
@@ -0,0 +1,55 @@
+
+
+## install.py
+`python install.py` should handle
+- install dependencies
+- download & preprocess dataset (if applicable)
+
+Note:
+- `torch` should not be a dependency, it has to work with latest pytorch master.
+
+## hubconf.py
+`hubconf.py` should contain a `class Model` with the following structure:
+
+```
+class Model:
+    def __init__(self, device=None, jit=False):
+        """ Required """
+        self.device = device
+        self.jit = jit
+
+    def get_module(self):
+        """ Required
+        Returns `model`, `example_input`
+        `model` should be torchscript model if self.jit is True.
+        Both `model` and `example_input` should be on `self.device` properly.
+        Both `model` and `example_input` can be lists/dicts as long as it's
+        consumable by train/eval function. """
+
+    def train(self):
```

Comment:
I think this should take an optional `niterations` which can be used to run a short warmup. It should run niteration passes of the training loop (i.e. forward/backward/optimizerstep). This will allow us to warmup the code by running niterations=1 in benchmarking.